check_api/src/main/java/com/google/errorprone/fixes/SuggestedFixes.java

```diff
@@ -982,6 +984,14 @@ private static boolean compilesWithFix(
     }
 
     JCCompilationUnit compilationUnit = (JCCompilationUnit) state.getPath().getCompilationUnit();
+    if (Options.instance(state.context).isSet(Option.VERBOSE)) {
```

Comment:
Based on @cushon's feedback in https://github.com/google/error-prone/pull/1473#discussion_r424096298 this should be changed to
```suggestion
    if (Options.instance(state.context).isSet("-verbose")) {
```
Will make that change if there's interest in getting this PR merged.